### PR TITLE
DEVDOCS-2034

### PR DESCRIPTION
**Pull request by mikrdgz**
# [DEVDOCS-2034](https://jira.bigcommerce.com/browse/DEVDOCS-2034)

## What changed?
* Adds IA changes to widgets overview

### CODE REVIEW

File: docs/api-docs/storefront/widgets/widgets-overview.md [hunk 1]
```diff
@@ -3,45 +3,47 @@
...
+[Regions](https://developer.bigcommerce.com/api-reference/storefront/widgets-api/regions/getcontentregions) are specific locations in the Stencil theme files where you can place a widget. A region is added at the file level using the format {{{region name="..."}}}. A region can be named however you like, but it is best practice to give it a name that is descriptive of the location and function. A theme file can have as many regions as you want, with more than one widget assigned to the region and the [Placement](/api-docs/storefront/widgets/widgets-overview#widgets_placements) `sort_order` controlling how the widgets appear on the storefront.
```
Comment (ektravel) [inline docs/api-docs/storefront/widgets/widgets-overview.md:30]: Looks like the Placements link needs to be updated. 
```suggestion
[Regions](https://developer.bigcommerce.com/api-reference/storefront/widgets-api/regions/getcontentregions) are specific locations in the Stencil theme files where you can place a widget. A region is added at the file level using the format {{{region name="..."}}}. A region can be named however you like, but it is best practice to give it a name that is descriptive of the location and function. A theme file can have as many regions as you want, with more than one widget assigned to the region and the [Placements](https://developer.bigcommerce.com/api-docs/store-management/widgets/overview#placements) `sort_order` controlling how the widgets appear on the storefront.
```

File: docs/api-docs/storefront/widgets/widgets-overview.md [hunk 2]
```diff
@@ -3,45 +3,47 @@
...
+Most themes in the BigCommerce marketplace come with predefined regions. It is best to use those first. By editing the theme and adding theme regions, updates will need to be managed manually.
 
-## Widget Templates
+## Widget templates
 
-[Widget Templates](/api-reference/storefront/widgets-api/widget-template/getwidgettemplates) are handlebars-enabled HTML templates which define the structure of the widget on a page. Widget Templates can use the built in handlebar helpers such as {{if}} and {{each}}. Using the helpers allows for conditional logic to be built into the Widget Templates. Widget templates can be reused to build multiple widgets.
+[Widget templates](/api-reference/storefront/widgets-api/widget-template/getwidgettemplates) are Handlebars-enabled HTML templates which define the structure of the widget on a page. Widget templates can use the built in handlebar helpers such as {{if}} and {{each}}. Using the helpers allows for conditional logic to be built into the Widget templates. Widget templates can be reused to build multiple widgets.
```
Comment (ektravel) [inline docs/api-docs/storefront/widgets/widgets-overview.md:40]:
```suggestion
[Widget templates](/api-reference/storefront/widgets-api/widget-template/getwidgettemplates) are Handlebars-enabled HTML templates which define the structure of the widget on a page. Widget templates can use the built-in Handlebars helpers such as {{if}} and {{each}}. Using the helpers allows for conditional logic to be built into the Widget templates. Widget templates can be reused to build multiple widgets.
```

File: docs/api-docs/storefront/widgets/widgets-overview.md [hunk 3]
```diff
@@ -3,45 +3,47 @@
...
 
-### Widget Template Examples
+### Widget template examples
 
-**Simple List**
+**Simple list**
 
-The Simple List template creates a list where each item in the list can have a different color. This example uses the [each block helper](https://handlebarsjs.com/api-reference/helpers.html) in handlebars to loop through each item in the list and display it. The text and color are determined by handlebars placeholders that are set when creating the [Widget](/api-reference/storefront/widgets-api/widget/createwidget).
+The simple list template creates a list where each item in the list can have a different color. This example uses the [each block helper](https://handlebarsjs.com/api-reference/helpers.html) in Handlebars to loop through each item in the list and display it. The text and color are determined by handlebars placeholders that are set when creating the [widget](/api-reference/storefront/widgets-api/widget/createwidget).
```
Comment (ektravel) [inline docs/api-docs/storefront/widgets/widgets-overview.md:46]:
```suggestion
The simple list template creates a list where each item in the list can have a different color. This example uses the [each block helper](https://handlebarsjs.com/api-reference/helpers.html) in Handlebars to loop through each item in the list and display it. The text and color are determined by Handlebars placeholders that are set when creating the [widget](/api-reference/storefront/widgets-api/widget/createwidget).
```

File: docs/api-docs/storefront/widgets/widgets-overview.md [hunk 4]
```diff
@@ -63,17 +65,17 @@ The slider takes advantage of loops to display each slide on the page.
...
-[Widgets](/api-reference/storefront/widgets-api/widget/getwidgets) are a unit of content that are placed on specific pages in a Stencil theme. Widgets are made of a widget configuration written in json and a widget template UUID. Widgets are rendered as part of the HTML on the storefront.
+[Widgets](/api-reference/storefront/widgets-api/widget/getwidgets) are a unit of content that are placed on specific pages in a Stencil theme. Widgets are made of a widget configuration written in JSON and a widget template UUID. Widgets are rendered as part of the HTML on the storefront.
 
 ![Widget](//s3.amazonaws.com/user-content.stoplight.io/6012/1551971053085 "Storefront Widget")
 
-### Widget Configuration
+### Widget configuration
 
-A Widget Configuration is the JSON payload that defines the content to be rendered with a given Widget Template. In the previous simple list example, the widget template loops over a list_items array using `{{#each}}`.
+A Widget configuration is the JSON payload that defines the content to be rendered with a given widget template. In the previous simple list example, the widget template loops over a list_items array using `{{#each}}`.
```
Comment (ektravel) [inline docs/api-docs/storefront/widgets/widgets-overview.md:74]:
```suggestion
A Widget configuration is the JSON payload that defines the content to be rendered with a given widget template. In the previous simple list example, the widget template loops over a `list_items` array using `{{#each}}`.
```

File: docs/api-docs/storefront/widgets/widgets-overview.md [hunk 5]
```diff
@@ -161,27 +163,27 @@ The example above uses `list_items_two`, on line four, in the configuration whic
 
 ## Placements
 
-[Placements](/api-reference/storefront/widgets-api/placement/createplacement) determine the Region where the Widget is placed and in what order. The order of the placement is controlled by the `sort_order` when creating the placement.
+[Placements](/api-reference/storefront/widgets-api/placement/createplacement) determine the region where the widget is placed and in what order. The order of the placement is controlled by the `sort_order` when creating the placement.
 A placement must be created in order to use a Widget on the storefront.
```
Comment (ektravel) [inline docs/api-docs/storefront/widgets/widgets-overview.md:167]:
```suggestion
A placement must be created in order to use a widget on the storefront.
```

File: docs/api-docs/storefront/widgets/widgets-overview.md [hunk 6]
```diff
@@ -107,14 +109,14 @@ In the example below, we define the list_items array and supply values for the c
...
-### Reusing Widget Configuration
+### Reusing widget configuration
 > Keep in mind that when reusing the `widget_configuration` the array will have to be called `list_items`. Otherwise, the widget will be created on the frontend, but there will be no data, so nothing is rendered. This is because `list_items` was defined when the widget template was originally created. The widget configuration name can be anything that is set when the widget_template is created.
 
 </div>
 </div>
 </div>
 
-**List Items Example: Incorrect**
+**List items example: incorrect**
```
Comment (uwikaiddi) [inline docs/api-docs/storefront/widgets/widgets-overview.md:119]:
```suggestion
**List items example: Incorrect**
```

File: docs/api-docs/storefront/widgets/widgets-overview.md [hunk 7]
```diff
@@ -138,7 +140,7 @@ In the example below, we define the list_items array and supply values for the c
 The example above uses `list_items_two`, on line four, in the configuration which is not the same as `list_items`. If `list_items` has already been established during the initial configuration it must be set as `list_items` when being used again.
 
 
-**List Items Example: Correct**
+**List items example: correct**
```
Comment (uwikaiddi) [inline docs/api-docs/storefront/widgets/widgets-overview.md:143]:
```suggestion
**List items example: Correct**
```

File: docs/api-docs/storefront/widgets/widgets-overview.md [hunk 8]
```diff
@@ -161,27 +163,27 @@ The example above uses `list_items_two`, on line four, in the configuration whic
...
 ## Placements
 
-[Placements](/api-reference/storefront/widgets-api/placement/createplacement) determine the Region where the Widget is placed and in what order. The order of the placement is controlled by the `sort_order` when creating the placement.
-A placement must be created in order to use a Widget on the storefront.
+[Placements](/api-reference/storefront/widgets-api/placement/createplacement) determine the region where the widget is placed and in what order. The order of the placement is controlled by the `sort_order` when creating the placement.
+A placement must be created in order to use a widget on the storefront.
 
 Placements can be used with `sort_order` and region to determine placement in a theme.
 
 ### Placements `entity_id`
```
Comment (uwikaiddi) [inline docs/api-docs/storefront/widgets/widgets-overview.md:171]:
```suggestion
### Placements entity_id
```

**Pull request merged**